src/VocabGeneration.test.js

```diff
@@ -17,7 +17,8 @@ const VERSION_RDFJS_IMPL = "^1.2.3";
 const NPM_REGISTRY = "http://localhost:4873";
 const RUN_NPM_INSTALL = false;
 const SUPPORT_BUNDLING = true;
-const PUBLISH_TO_REPO_LIST = ["mavenLocal", "npmLocal"];
+// const PUBLISH_TO_REPO_LIST = ["mavenLocal", "npmLocal"];
+const PUBLISH_TO_REPO_LIST = ["npmLocal"];
```

Comment:
Why keep the commented out line?

Reply:
SImply 'cos all it needs to have local Maven running again is having a copy of the Java vocab-term library available in the local `.m2` cache (or on Maven Central or wherever, of course!).